src/components/ForgotPassword/LockIcon.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+function Icon() {
+   return (
+      <svg
+         xmlns="http://www.w3.org/2000/svg"
+         width="171"
+         height="214"
+         fill="none"
+         viewBox="0 0 171 214"
+      >
+         <path
+            fill="#024E6B"
+            d="M85.333 117.333a15.891 15.891 0 00-15.027 10.206 15.895 15.895 0 004.36 17.634V160a10.665 10.665 0 0010.667 10.667A10.665 10.665 0 0096 160v-14.827a15.894 15.894 0 00-10.667-27.84zm53.334-42.666V53.333a53.333 53.333 0 10-106.667 0v21.334a32 32 0 00-32 32v74.666a32.001 32.001 0 0032 32h106.667a31.999 31.999 0 0032-32v-74.666a32.001 32.001 0 00-32-32zM53.333 53.333a32 32 0 0164 0v21.334h-64V53.333zm96 128A10.666 10.666 0 01138.667 192H32a10.668 10.668 0 01-10.667-10.667v-74.666A10.668 10.668 0 0132 96h106.667a10.665 10.665 0 0110.666 10.667v74.666z"
+         />
+      </svg>
+   );
+}
+
+export default Icon;
```

Comment:
Don't think you need to make this a separate .jsx file. You should be able to just import the svg.